10LessonTen/Checkpoint2/css/style.css

```diff
@@ -0,0 +1,281 @@
+body {
+  margin: 0;
+  font-family: 'Open Sans', sans-serif;
+}
+
+#navigation {
+  height:40px;
+  background-color: #d0285b;
+  margin: 0;
+}
+
+#navtitle {
+  font-size: 20px;
+  font-weight: 400;
+  padding: 0;
+  display:inline-block;
+  margin: 7px 0 0 5px;
+  color: white;
+}
+
+ul {
+  list-style-type: none;
+  width: 50%;
+  height: 40px;
+  margin: 0;
+  padding: 0;
+  display:inline-block;
+  float: right;
+  margin-top: 12px;
+}
+
+ul * {
+  list-style-type: none;
+  text-decoration: none;
+  color: white;
+  font-weight: 100;
+  font-size: 12px;
+}
+
+li {
+  float: right;
+  padding-right:20px;
+}
+
+/*TITLE CARD STARTS HERE */
+
+#titlecard {
+  background-image: url(../img/kangaroo1.jpg);
+  width: 100%;
```

Comment:
this makes sense! 100% width, no padding = 100%. further down this css, there's 100vw for your sections and footer that interacting with other elements to create > 100% of the screen = side scroll = jake mad = jake crush. be consistent with your widths of sections.